data/json/npcs/BGSS_talk.json

```diff
@@ -0,0 +1,922 @@
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+ },
+  {
+    "id": "BGSS_LOST_PARTNER_1_STORY2",
+    "type": "talk_topic",
+    "dynamic_line": "That's how it goes, you know? These are the end times. I don't really want to talk about it more than that. And honestly, I never really felt like I belonged, in the old world. In a weird way, I actually feel like I have a purpose now. Do you ever get that?",
+    "responses": [
+      { "text": "No, that's messed up.", "topic": "BGSS_LOST_PARTNER_1_NOWAY" },
+      { "text": "Yeah, I get that. Sometimes I feel like my existence began shortly after the cataclysm.", "topic": "BGSS_LOST_PARTNER_1_YEAH" },
+      { "text": "Tell me about those plant monsters.", "topic": "BGSS_LOST_PARTNER_1_TRIFFIDS" },
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+ },
+  {
+    "id": "BGSS_LOST_PARTNER_1_YEAH",
+    "type": "talk_topic",
+    "dynamic_line": "I guess those of us who made it this far have to have made it for a reason, or something. I don't mean like a religious reason, just... we're survivors.",
+    "responses": [
+      { "text": "Tell me about those plant monsters.", "topic": "BGSS_LOST_PARTNER_1_TRIFFIDS" },
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+ },
+  {
+    "id": "BGSS_LOST_PARTNER_1_NOWAY",
+    "type": "talk_topic",
+    "dynamic_line": "Haha, yeah, I can see why you'd think that. I don't mean it's a good apocalypse. I just mean that at least now I know what I'm doing every day. I'd still kill a hundred zombies for an internet connection and a night watching crappy movies with... sorry. Let's change the subject.",
+    "responses": [
+      { "text": "Tell me about those plant monsters.", "topic": "BGSS_LOST_PARTNER_1_TRIFFIDS" },
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+ },
+  {
+    "id": "BGSS_LOST_PARTNER_1_TRIFFIDS",
+    "type": "talk_topic",
+    "dynamic_line": "Yeah, have you seen them yet? They're these awful walking flowers with a big fucking stinger in the middle. They travel in packs. They hate the zombies, and we were using them for cover to clear a horde of the dead. Unfortunately, turns out the plants are better trackers than the zombies. They almost seemed intelligent... I barely made it out, only because they were, uh, distracted.",
+    "responses": [
+      { "text": "I'm sorry you lost someone.", "topic": "BGSS_LOST_PARTNER_1_STORY2" },
+      { "text": "Let's talk about something else.", "topic": "TALK_FRIEND" }
+    ]
+ },
+
+
+  {
+    "id": "BGSS_LOST_PARTNER_2_STORY1",
+    "type": "talk_topic",
+    "dynamic_line": "Just another shitty tale of love and loss. Not something I like to tell.",
+    "responses": [
+       { "text": "It might help to get it off your chest.", 
+        "trial": { "type": "PERSUADE", "difficulty": 70, "mod": [ [ "TRUST", 3 ] ] },
```

Comment:
difficulty 70 is extremely hard.

Reply:
Woops, I forgot to playtest that with my speech skill set lower than 20.